Tasks/src/Enums/Figure1.java

```diff
@@ -0,0 +1,17 @@
+package Enums;
+
+enum Figure1 {
```

Comment:
@KroosT note: strive to not use numbers in class names like this: `Figure1` and `Figure2`.
you could put these two enums in different packages